srcpkgs/nxt/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+In order to run `nxt-tor`, TOR needs to be running.
+In order to install TOR, run `xbps-install -S tor` and then preferably start the tor service.
```

Comment:
This isn't required, put a dependency on our tor runit service instead.

```
sv check tor >/dev/null || exit 1
```

Reply:
please ping me when the proposed changes are done so i can merge this quickly

Reply:
@maxice8 I finally did the update. (sorry for the wait, didn't want to rebase the branch for some time to get it work again)
I'll also update it to the next version in the next minutes, since the current version is deprecated.